src/google-maps/google-map/google-map.ts

```diff
@@ -74,7 +65,13 @@ export class GoogleMap implements OnChanges, OnInit, OnDestroy {
   private readonly _zoom = new BehaviorSubject<number|undefined>(undefined);
   private readonly _destroy = new Subject<void>();
   private _mapEl: HTMLElement;
-  _googleMap: UpdatedGoogleMap;
+
+  /**
+   * The underlying google.maps.Map object
+   *
+   * See developers.google.com/maps/documentation/javascript/reference/map#Map
+   */
+  googleMap?: google.maps.Map;
```

Comment:
Since the object is created asynchronously, maybe we should expose an Observable or a Promise instead?

Reply:
Possibly, I'm unsure if there's actually a case where the component is initialized but the google map object is unset outside of the server-side rendering case. I do not like the idea of an observable wrapped around a union of the map and undefined though. We could have the observable never emit if we are server-side rendering. Any thoughts, @jelbourn ?

Reply:
Can the map ever be initialized by `ngOnInit`? If so, what would happen? If there's no situation where the map can be undefined without throwing an error by the end of `ngOnInit`, I don't see the need to make it a promise/stream.

Reply:
The components were changed so that when they're loaded server side, the map is not initialized. However, in that case, the map is never initialized. To me, that seems like undefined would be more appropriate than an unemitted observable.